### ISSUE TITLE
gnt-network info / list fails when reading retrieving external reservations

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#379]
Originally reported of Google Code with ID 323.

```
Using "gnt-network info" or "gnt-network list" (to retrieve the external reservations) fails with an uncaught exception.

> gnt-network list
Network  Subnet         Gateway NetworkType MacPrefix GroupList Tags
mynet 192.168.0.0/16 None    None        None                
> gnt-network info mynet
Unhandled protocol error while talking to the master daemon:
Caught exception: _GetNetworkStatsField() takes exactly 3 arguments (4 given)
> gnt-network list -o external_reservations mynet
Unhandled protocol error while talking to the master daemon:
Caught exception: _GetNetworkStatsField() takes exactly 3 arguments (4 given)

The net was created with:
gnt-network add --network=192.168.0.0/16 --add-reserved-ips=192.168.0.5 mynet

```
Originally added on 2012-11-27 08:55:17 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-11-27 18:06:50 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
This has been released in 2.7
```
Originally added on 2013-05-09 05:44:21 +0000 UTC.

Changed State: Released
Added to Milestone: Release2.7
